patch-package
--- a/node_modules/@cycle/history/lib/cjs/captureClicks.js
+++ b/node_modules/@cycle/history/lib/cjs/captureClicks.js
@@ -17,6 +17,9 @@ function sameOrigin(href) {
     }
     return href && href.indexOf(window.location.origin) === 0;
 }
+
+var baseHref = document.querySelector('base').getAttribute('href');
+
 function makeClickListener(push) {
     return function clickListener(event) {
         if (which(event) !== 1) {
@@ -51,6 +54,11 @@ function makeClickListener(push) {
         }
         event.preventDefault();
         var pathname = element.pathname, search = element.search, _a = element.hash, hash = _a === void 0 ? '' : _a;
+
+        // strip base href from the pathname. it gets added back by the history driver, which results
+        // in having it twice without this fix.
+        if (pathname.indexOf(baseHref) == 0) pathname = pathname.substr(baseHref.length-1)
+
         push(pathname + search + hash);
     };
 }
